Date_05/main.rb

```diff
@@ -0,0 +1,63 @@
+load 'customer.rb'
+require 'date'
+# require_relative 'connect_DB'
+
+class Main 
+  def insertCustomer
+    $cus_id = $client.query("SELECT MAX(cus_id) FROM customers").first["MAX(cus_id)"]   
+    @cus = Customer.new
+    @cus.input 
+    $client.query("insert into customers(name, gender, join_date) values ('#{@cus.username}', '#{@cus.gender}', '#{@cus.join_date}')")
+    print "Your ID is : #{$cus_id}\n" 
+  end 
+
+  def insertBill
+    @cus = Customer.new
+    @cus.buy($cus_id, $product_name, $quantity)
+    $client.query("insert into bills(cus_id, pro_name, quanlity) values (#{$cus_id.to_i}, '#{$product_name}', #{$quantity.to_i})")
+  end
+
+  def showProduct
```

Comment:
Use `snake_case` to define method
Define method `show_product` in Customer class to insert into DB.